src/config.ts

```diff
@@ -22,10 +25,25 @@ export interface Config {
    * default: ['/']
    */
   validatePaths: string[];
+
   /**
    * Optional base path to swagger ui bundle
    */
   swaggerUiBundleBasePath: string;
+
+  /**
+   * Optional custom error handler
+   */
+  errorHandler: (error: Error, ctx: Context)=> void,
+}
+
+function defaultErrorHandler(err: Error, ctx: Context) {
+  if (err instanceof ChowError) {
+    const json = err.toJSON();
+    ctx.throw(400, 'Request validation error', { expose: true, ...json });
```

Comment:
Sorry, didn't notice the message only valid for Request, not Response. Hmm...

Reply:
Have updated this.